Makefile

```diff
@@ -1045,13 +1046,23 @@ ifeq ($(XZ), 0)
 endif
 
 .PHONY: bench-all
-bench-all:
+bench-all: bench-addons
 	@echo "Please use benchmark/run.js or benchmark/compare.js to run the benchmarks."
 
 .PHONY: bench
-bench:
+bench: bench-addons
```

Comment:
Not a blocking objection, but:

Currently, all `make bench` does is print a message telling people to run the tests manually instead, right?

This will build the addons and then tell the person to run their benchmarks manually?

Would it make more sense to not do that and just have people know they need to run `make bench-addons` if they are running one of the two benchmarks that require it? No one runs `make bench` anyway unless they stumble upon it. Compiling stuff and then printing a message saying, basically, "don't use this command" seems like something that could be improved.